model/message_export.go

```diff
@@ -20,6 +20,7 @@ type MessageExport struct {
 
 	PostId         *string
 	PostCreateAt   *int64
+	PostUpdateAt   *int64
```

Comment:
I added this field because it is used in the enterprise repo: https://github.com/mattermost/enterprise/pull/491/files#diff-7e66047c8d790ba68cbb3cb2903abd64R191